Upstream: yes

From fd584ec6707ae5b6e7b07fe19443b513fb0ba62b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ludovic=20Court=C3=A8s?= <ludo@gnu.org>
Date: Thu, 24 Apr 2014 23:46:20 +0200
Subject: [PATCH 2/2] build: Clean up rules for meta/{guild,guile-config}.

* meta/Makefile.am (guile-config, guild): Avoid useless 'cat'
  invocation'.  Quote all occurrences of '@'.
  Suggested by Eric Blake <eblake@redhat.com>.
---
 meta/Makefile.am | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/meta/Makefile.am b/meta/Makefile.am
index 102b797..57644bc 100644
--- a/meta/Makefile.am
+++ b/meta/Makefile.am
@@ -46,21 +46,21 @@ aclocal_DATA = guile.m4
 
 guile-config: $(srcdir)/guile-config.in $(top_builddir)/config.status
 	guile="@bindir@/`echo guile | $(SED) -e '$(program_transform_name)'`" ;	\
-	cat $(srcdir)/guile-config.in						\
-	| $(SED) -e "s,@pkgconfigdir@,$(pkgconfigdir),g"			\
-		 -e "s,@""PKG_CONFIG@,$(PKG_CONFIG),g"				\
-		 -e "s,@installed_guile@,$$guile,g"				\
-	> guile-config.out
-	mv guile-config.out guile-config
-	chmod +x guile-config
+	$(SED) -e "s,[@]pkgconfigdir[@],$(pkgconfigdir),g"			\
+	       -e "s,[@]PKG_CONFIG[@],$(PKG_CONFIG),g"				\
+	       -e "s,[@]installed_guile[@],$$guile,g"				\
+	  < "$(srcdir)/guile-config.in"						\
+	  > "$@.out"
+	chmod +x "$@.out"
+	mv "$@.out" "$@"
 
 guild: $(srcdir)/guild.in $(top_builddir)/config.status
 	guile="@bindir@/`echo guile | $(SED) -e '$(program_transform_name)'`" ;	\
-	cat $(srcdir)/guild.in							\
-	| $(SED) -e "s,@installed_guile@,$$guile,g"				\
-	         -e "s,[@]PACKAGE_NAME[@],$(PACKAGE_NAME),g"			\
-	         -e "s,[@]PACKAGE_BUGREPORT[@],$(PACKAGE_BUGREPORT),g"		\
-	> "$@.out"
+	$(SED) -e "s,[@]installed_guile[@],$$guile,g"				\
+	       -e "s,[@]PACKAGE_NAME[@],$(PACKAGE_NAME),g"			\
+	       -e "s,[@]PACKAGE_BUGREPORT[@],$(PACKAGE_BUGREPORT),g"		\
+	  < "$(srcdir)/guild.in"						\
+	  > "$@.out"
 	chmod +x "$@.out"
 	mv "$@.out" "$@"
 
-- 
2.1.2

